src/main/java/com/example/techiteasybackend/models/Television.java

```diff
@@ -0,0 +1,210 @@
+package com.example.techiteasybackend.models;
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "Televisions")
+public class Television {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    private AvailableSize availableSize;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+    private LocalDate sellDate;
+
+    public enum AvailableSize {
```

Comment:
Dit kan hier, maar het is netter om hier een eigen classe van te maken, dan maak je dus nog een mapje in models aan die enums heet en dan kan je daar de class in maken.